### PR TITLE
feat: Add a startMaintenance and stopMaintenance

### LINKED ISSUE #2454: Add a startMaintenance and stopMaintenance automated task in `rotate-with-dkg` to prevent L1 conflict on busy chains that you'd like to rotate
We've been able to get away with rotating without going into maintenance mode until now, but with the busy testnet this is no longer possible. I can manually put the chain in maintenance mode, rotate, and get it out of maintenance mode again. It would be nice to do this by default in the `rotate-with-dkg` command, optionally with a flag to disable this behavior.

### PR DESCRIPTION
**Pull request by howjmay**
close #2454

### CODE REVIEW

File: tools/wasp-cli/chain/rotate.go [hunk 1]
```diff
@@ -115,3 +120,22 @@ func rotateTo(chain string, newStateControllerAddr iotago.Address) {
 	log.Check(err)
 	fmt.Fprintf(os.Stdout, "Chain rotation transaction issued successfully.\nTXID: %s\n", txID.ToHex())
 }
+
+func setMaintenanceStatus(chain, node string, start bool) {
```
Comment (kape1395) [inline tools/wasp-cli/chain/rotate.go:124]: Have you tested, if that actually works?

Comment (howjmay) [reply tools/wasp-cli/chain/rotate.go:124]: Here is how I tested and the result. Result is the returned value of `GetMaintenanceStatus` call

```
 wasp-cli chain rotate-with-dkg --skip-version-check
Posted off-ledger request (check result with: wasp-cli chain request 0x08b2b62b78824d8573d7242f5d7426b7b88019d4fbcd88ef94c372ef1eca92b20000)
result:  {"Items":[{"key":"0x6d","value":"0x01"}]}
DKG successful
Address: tst1qqnjuk7duyh8vq0mggagfqs2tfrq4lt8y926vg4xh5jtrahh95exc9kyzrz
* committee size = 1
* quorum = 1
* members: 0xc531eee2e923dee8bfc83ea40114c0b7c6d1c7bf9553426954ec9ce88ea91b25 (me)

2023-05-11T00:00:31-06:00       INFO    nc      Posted blockID 0xa9ff2d355b4fa8c2a9e8837f4dab49164e7ba2912c6380df1b27ac4f8951764b
2023-05-11T00:00:31-06:00       INFO    nc      Posted transaction id 0x3ff380ce948c564704a978bbe67a244f06946cd18ec1e917e69c231a7132a891
Chain rotation transaction issued successfully.
TXID: 0x3ff380ce948c564704a978bbe67a244f06946cd18ec1e917e69c231a7132a891
Posted off-ledger request (check result with: wasp-cli chain request 0xbd7ee3aae9f82f8c3bb0e2f0710909803d7b87cb149d0e98cd8d0f34fee468200000)
result:  {"Items":[{"key":"0x6d","value":"0x00"}]}
```

Comment (howjmay) [reply tools/wasp-cli/chain/rotate.go:124]: I saw the maintenance status switched successfully

**Review by jorgemmsilva — APPROVED** [review]
lgtm, please make sure these tests still pass before merging:
https://github.com/iotaledger/wasp/blob/develop/tools/cluster/tests/wasp-cli_rotation_test.go
(don't forget to `make install` beforehand :stuck_out_tongue: )

**Pull request merged**